### PR TITLE
Mic-4911/update components for lookup configuration

### PR DESCRIPTION
**Pull request by albrja**
### Updates remaining components for lookup table configuration updates.
- *Category*: Refactor
- *JIRA issue*: [MIC-4911](https://jira.ihme.washington.edu/browse/MIC-4911)
- *Research reference*: 

### Changes and notes
-updates components for lookup configuration 
-adds new distribution subclass

### Verification and Testing
Successfully ran full simulation

### CODE REVIEW

**Comment by patricktnast** [comment]
can you lint this real quick before I review?

File: src/vivarium_gates_nutrition_optimization_child/components/wasting.py [hunk 1]
```diff
@@ -24,14 +24,19 @@
...
     def time_step_prepare_priority(self) -> int:
         # we want to reset propensities before updating previous state column
         return 4
 
-    @property
-    def name(self) -> str:
-        return f"wasting_treatment_{self.risk}"
+    # @property
+    # def name(self) -> str:
+    #     return f"wasting_treatment_{self.risk}"
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/components/wasting.py:39]: Can we remove this commented code?

File: src/vivarium_gates_nutrition_optimization_child/constants/data_keys.py [hunk 2]
```diff
@@ -388,9 +388,9 @@ class __LowBirthWeightShortGestation(NamedTuple):
         "risk_factor.low_birth_weight_and_short_gestation.relative_risk_interpolator"
     )
 
-    PAF: TargetString = (
-        "risk_factor.low_birth_weight_and_short_gestation.population_attributable_fraction"
-    )
+    #    PAF: TargetString = (
+    #        "risk_factor.low_birth_weight_and_short_gestation.population_attributable_fraction"
+    #    )
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/constants/data_keys.py:393]: Is this needed?

File: src/vivarium_gates_nutrition_optimization_child/data/lbwsg_paf.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ components:
 configuration:
     input_data:
         input_draw_number: 0
-        artifact_path: '/mnt/team/simulation_science/pub/models/vivarium_gates_nutrition_optimization_child/artifacts/model13.0/ethiopia.hdf'
+        artifact_path: '/mnt/team/simulation_science/pub/models/vivarium_gates_nutrition_optimization_child/artifacts/subnational/pakistan.hdf'
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/data/lbwsg_paf.yaml:17]: Is this change intentional?

File: src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml [hunk 4]
```diff
@@ -15,9 +15,9 @@ components:
             - RiskEffect('risk_factor.moderate_acute_malnutrition_treatment', 'risk_factor.better_moderate_acute_malnutrition_to_mild_child_wasting.transition_rate')
             - RiskEffect('risk_factor.moderate_acute_malnutrition_treatment', 'risk_factor.worse_moderate_acute_malnutrition_to_mild_child_wasting.transition_rate')
 
-            - LBWSGRiskEffect('cause.diarrheal_diseases.excess_mortality_rate')
-            - LBWSGRiskEffect('cause.lower_respiratory_infections.excess_mortality_rate')
-            - LBWSGRiskEffect('cause.affected_unmodeled.cause_specific_mortality_rate')
+            # - LBWSGRiskEffect('cause.diarrheal_diseases.excess_mortality_rate')
+            # - LBWSGRiskEffect('cause.lower_respiratory_infections.excess_mortality_rate')
+            # - LBWSGRiskEffect('cause.affected_unmodeled.cause_specific_mortality_rate')
```
Comment (rmudambi) [inline src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml:20]: Should these be commented out?

**Review by rmudambi — APPROVED** [review]

File: src/vivarium_gates_nutrition_optimization_child/components/__init__.py [hunk 5]
```diff
@@ -2,6 +2,9 @@
     RiskAttributableDisease,
     SIS_with_birth_prevalence,
 )
+from vivarium_gates_nutrition_optimization_child.components.distribution import (
+    PolytomousDistribution,
```
[the remark this answers is not in the document]
Comment (albrja) [reply src/vivarium_gates_nutrition_optimization_child/components/__init__.py:6]: So we can import it.

File: src/vivarium_gates_nutrition_optimization_child/data/lbwsg_paf.yaml (hunk 3 above)
Comment (albrja) [reply src/vivarium_gates_nutrition_optimization_child/data/lbwsg_paf.yaml:17]: This is so I don't forget later.

File: src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml (hunk 4 above)
Comment (albrja) [reply src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml:20]: Yes we haven't run the paf sim yet.
